### PR TITLE
Updating Genie Quick Trigger documentation with yang action

### PR DESCRIPTION
**Pull request by miott**
Updating Genie Quick Trigger documentation with yang action

### CODE REVIEW

**Comment by jeaubin** [comment]
Hi @miott  I asked @danielgraziano  to go over it as he used it last week.

I think we should have more example, it could be in a different file (The examples) so it doesnt make this page too busy, but it would be very useful as yang can be confusing.

**Comment by miott** [comment]
> Hi @miott I asked @danielgraziano to go over it as he used it last week.
> 
> I think we should have more example, it could be in a different file (The examples) so it doesnt make this page too busy, but it would be very useful as yang can be confusing.

I found an "examples" repository. I could make a quicktrigger directory and create a job_quicktrigger_yang.py and quicktrigger_yang.yml.  Is that what you were thinking?

**Comment by miott** [comment]
Added a "yang.rst" page next to writeblitz.rst and reformatted.

File: develop-guide/writeblitz/writeblitz.rst [hunk 1]
```diff
@@ -364,7 +364,28 @@ ______
 yang
 ____
 
-Documentation in development
+The ``yang`` action is designed to work with differring underlying protocols, but, at the
```
Comment (jeaubin) [inline develop-guide/writeblitz/writeblitz.rst:367]: can you add a link to your more in details documentation

**Comment by jeaubin** [comment]
1 comment then let's merge it and i'll release it

**Comment by jeaubin** [comment]
Thanks @miott

**Pull request merged**